satpy/tests/reader_tests/conftest.py

```diff
@@ -15,4 +15,242 @@
+    ds[f"data/navigation/{prefix}satellite_zenith_angle"] = make_fake_angles(geo_size, geo_dims)
+    ds["status/satellite/subsat_latitude_end"] = np.array(22.39)
+    ds["status/satellite/subsat_longitude_start"] = np.array(304.79)
+    ds["status/satellite/subsat_latitude_start"] = np.array(55.41)
+    ds["status/satellite/subsat_longitude_end"] = np.array(296.79)
+
+    return ds
+
+@pytest.fixture(scope="module")
+def eps_sterna_mwr_file(tmp_path_factory, fake_mwr_data_array):
+    """Create an EPS-Sterna MWR l1b file."""
+    ds = aws_eps_sterna_mwr_l1bfile(fake_mwr_data_array, eps_sterna=True)
+
+    tmp_dir = tmp_path_factory.mktemp("eps_sterna_mwr_l1b_tests")
+    start_time = dt.datetime.fromisoformat(ds.attrs["sensing_start_time_utc"])
+    end_time = dt.datetime.fromisoformat(ds.attrs["sensing_end_time_utc"])
+    platform_name = "AWS1"
+    processing_time = random_date(dt.datetime(2024, 9, 1, 13), dt.datetime(2030, 6, 1))
+    filename = tmp_dir / compose(file_pattern, dict(country="XX",
+                                                    organisation="EUMETSAT",
+                                                    location="Darmstadt",
+                                                    processing_level="1B",
+                                                    originator="EUMT",
+                                                    start_time=start_time, end_time=end_time,
+                                                    processing_time=processing_time,
+                                                    platform_name=platform_name))
+    ds.to_netcdf(filename)
+    return filename
+
+@pytest.fixture(scope="module")
+def aws_mwr_file(tmp_path_factory, fake_mwr_data_array):
+    """Create an AWS MWR l1b file."""
+    ds = aws_eps_sterna_mwr_l1bfile(fake_mwr_data_array, eps_sterna=False)
+
+    tmp_dir = tmp_path_factory.mktemp("aws_l1b_tests")
+    start_time = dt.datetime.fromisoformat(ds.attrs["sensing_start_time_utc"])
+    end_time = dt.datetime.fromisoformat(ds.attrs["sensing_end_time_utc"])
+    platform_name = "AWS1"
+    processing_time = random_date(dt.datetime(2024, 9, 1, 13), dt.datetime(2030, 6, 1))
+    filename = tmp_dir / compose(file_pattern, dict(country="SE",
+                                                    organisation="SMHI",
+                                                    location="Norrkoping",
+                                                    processing_level="1B",
+                                                    originator="SMHI",
+                                                    start_time=start_time, end_time=end_time,
+                                                    processing_time=processing_time,
+                                                    platform_name=platform_name))
+    ds.to_netcdf(filename)
+    return filename
+
+
+@pytest.fixture(scope="module")
+def aws_mwr_l1c_file(tmp_path_factory, fake_mwr_data_array):
```

Comment:
A lot of this fixture is identical to `aws_eps_sterna_mwr_l1bfile()` function, so maybe that could be used to get the common parts and only update the parts that are different?

Reply:
As the level-1c file format is not frozen and not documented (actually same can be said about the ESA AWS level-1b format). I would rather leave it as is for now.